packages/amplication-server/package.json

```diff
@@ -36,6 +36,7 @@
   "dependencies": {
     "@codebrew/nestjs-storage": "^0.1.4",
     "@extra-set/difference": "^2.1.52",
+    "@google-cloud/cloudbuild": "^2.0.3",
```

Comment:
The deploy service should be created in a separate package consumed by amplication-server
This new service should use a new interface with different implementations for different cloud providers (currently only with GCP)

Reply:
It is implemented that way. I'll extract the code to another package.